test_samples/src/lib.rs

```diff
@@ -0,0 +1,43 @@
+//! Utility crate for standardized and random signatories.
+
+use iroha_crypto::{Algorithm, KeyPair, PrivateKey};
+use iroha_data_model::prelude::AccountId;
+use once_cell::sync::Lazy;
+
+/// Generate [`AccountId`](iroha_data_model::account::AccountId) in the given `domain`.
+///
+/// # Panics
+///
+/// Panics if the given `domain` is invalid as [`Name`](iroha_data_model::name::Name).
+#[cfg(feature = "rand")]
+pub fn gen_account_in(domain: impl core::fmt::Display) -> (AccountId, KeyPair) {
+    let key_pair = KeyPair::random();
+    let account_id = format!("{}@{}", key_pair.public_key(), domain)
+        .parse()
+        .expect("domain name should be valid");
+    (account_id, key_pair)
+}
+
+macro_rules! static_signatory_ed25519 {
```

Comment:
I would split that into two macros with clearer naming: `declare_key_pair` and `declare_account_with_key_pair`.

Reply:
doesn't matter right now because it's just in tests